src/stop.sh

```diff
@@ -104,8 +104,11 @@ do_umountall() {
     done
 
     #Get fuse mounts of the cluster
-    CEPH_FUSE_MNTS=$("${CEPH_BIN}"/ceph -c $conf_fn tell mds.* client ls 2>/dev/null | grep mount_point | tr -d '",' | awk '{print $2}')
-    [ -n "$CEPH_FUSE_MNTS" ] && sudo umount -f $CEPH_FUSE_MNTS
+    num_of_ceph_mdss=$(ps -e | grep ceph-mds | wc -l)
```

Comment:
- [ I feel we should strictly check only `ceph-mds` process.
Here if some process say `ceph-mds-util`, `ceph-mds-runner` etc exist then this condition will be true. (I believe this is not desired)]